### PR TITLE
Added Palindrome number code using Java 

### LINKED ISSUE #1660: Palindrome Number
**Is your feature request related to a problem? Please describe.**
Given an integer x, return true if x is palindrome integer

**Describe the solution you'd like**
I would like to add code in code folder in c++ .

**Do you want to work on it**
Yes
Want to add code under GSSoC'21 participant .

### PR DESCRIPTION
**Pull request by Tejal18**
This code checks whether a number is palindrome number or not.
I have added the algorithm and code for this. 

**Fixes #1660 .**

Please do review my code @rish-singhal @tarun26091999  @raksha009 .
Thanks.

### CODE REVIEW

File: Code/Java/Palindrome_number.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/* Check for Palindrome Number */
```
Comment (raksha009) [inline Code/Java/Palindrome_number.java:1]: Include line 1 to 12 in a single multiline comment.

File: Code/Java/Palindrome_number.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Step 5: If both numbers are same, return true
+Step 6: Else return false */
+
+package palin;
+import java.util.*;
+class P
+{
+	boolean Panlindrome (int n)
+	{
+		int rem=0;  //for remainder
```
Comment (raksha009) [inline Code/Java/Palindrome_number.java:20]: Add comments above the line throughout yr code.

File: Code/Java/Palindrome_number.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		if(sum==temp) {   //original number is equal to reversed number 
+			return true;  //true if number is a palindrome
+		}
+		else 
+		{
+			return false; //false if number is not a palindrom
+		}
+	}
+}
+public class Pmain {
```
Comment (raksha009) [inline Code/Java/Palindrome_number.java:38]: class name should be same as file name.

File: Code/Java/Palindrome_number.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
```
Comment (raksha009) [inline Code/Java/Palindrome_number.java:41]: Remove this line

File: Code/Java/Palindrome_number.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+		P pa = new P();  //create object of class P
+		System.out.println("-------To check whether a number is palindrome or not-------");
+		System.out.println("Enter a number: ");
+		n1 = sc.nextInt();
+		System.out.println("Input: "+n1);
+		result= pa.Panlindrome(n1);
+		System.out.println("Output: "+result);
+	}
+}
+/*The time complexity to check if a number is a palindrome or not is  O(log10​(n)), n is the input number
```
Comment (raksha009) [inline Code/Java/Palindrome_number.java:54]: Include line 54 till end in a single multiline comment.

File: Code/Java/Palindrome_number.java [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+Input: 124
+Output: false
+
+Test case 2:
+-------To check whether a number is palindrome or not-------
+Enter a number: 
+656
+Input: 656
+Output: true
+
```
Comment (raksha009) [inline Code/Java/Palindrome_number.java:71]: Add time complexity here and follow the pattern 
Time Complexity:  ______   where n

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Comment by Tejal18** [comment]
@raksha009 I have updated the changes which you have suggested . Please check it once. Thanks.

**Review by raksha009 — APPROVED** [review]

**Pull request merged**